### ISSUE TITLE
Error trying to getSources() in RN 0.49

### ISSUE THREAD

**Issue by 1mike12** [oney/react-native-webrtc#361]
I've built the example app fine on my system, and I've read from issues that RN 0.47 is supported. 

```
Attempt to invoke interface method 'void com.facebook.react.bridge.Callback.invoke(java.lang.Object[])' on a null object reference
mediaStreamTrackGetSources
    WebRTCModule.java:474
invoke
    Method.java
invoke
    JavaMethodWrapper.java:363
invoke
    JavaModuleWrapper.java:162
run
    NativeRunnable.java
handleCallback
    Handler.java:751
dispatchMessage
    Handler.java:95
dispatchMessage
    MessageQueueThreadHandler.java:31
loop
    Looper.java:154
run
    MessageQueueThreadImpl.java:194
run
    Thread.java:761

```
#### Platform information
RN 0.49
* **Plugin version**:
^1.58.2
* **OS**: 
android
* **OS version**:
7.1

**Comment by 1mike12** [comment]
Just realized that maybe this is just a documentation error? Is the method supposed to be thenable? Looks like in java land `Callback callback` is obviously null since I don't pass one in. 

the part i copied from docs 
```javascript
MediaStreamTrack
  .getSources()
  .then(sourceInfos => {
    console.log(sourceInfos);
    let videoSourceId;
    for (const i = 0; i < sourceInfos.length; i++) {
      const sourceInfo = sourceInfos[i];
      if(sourceInfo.kind == "video" && sourceInfo.facing == (isFront ? "front" : "back")) {
        videoSourceId = sourceInfo.id;
      }
    }
```
works fine if I do it similar to the example app
```javascript
MediaStreamTrack
  .getSources(sourceInfos => {
    console.log(sourceInfos);
    let videoSourceId;
    for (const i = 0; i < sourceInfos.length; i++) {
      const sourceInfo = sourceInfos[i];
      if(sourceInfo.kind == "video" && sourceInfo.facing == (isFront ? "front" : "back")) {
        videoSourceId = sourceInfo.id;
      }
    })

```

**Comment by zxcpoiu** [comment]
@1mike12 

I guess it's my bad that not release a new version but the `Readme` changed.
promise support only on the latest master. ( I've released a 1.58.3 including promise support and #360 )

can you try on 1.58.3?

**Comment by 1mike12** [comment]
yup that worked. Thanks for quick reply
